rpc/params.go

```diff
@@ -4,8 +4,8 @@ import sdk "github.com/irisnet/irishub-sdk-go/types"
 
 type Params interface {
 	sdk.Module
-	QueryParams(subspace, key string) (SubspaceParamsResponse, sdk.Error)
-	//QueryParams(module string) (SubspaceParamsResponse, sdk.Error)
+	QueryParamsBySubAndKey(subspace, key string) (SubspaceParamsResponse, sdk.Error)
```

Comment:
why there are two method for query params?

Reply:
Sorry, I just submitted the code is incompleted code, here are two queries, one is `QueryParamsBySubAndKey(subspace, key string)`
ther othrer is `QueryParams(moduleName string)`
I' will submit another pr later